rasa/cli/scaffold.py

```diff
@@ -220,6 +220,9 @@ def run(args: argparse.Namespace) -> None:
         # Can't use `if not path` either, as `None` will be handled differently (abort)
         if path == "":
             path = "."
+        if path[0] == "~":
```

Comment:
There is a comment above the previous if statement warning against using `if not path:` since if path is `None`, the handling should be different.
However, this change would crash in case `path` is `None`. Either the comment is wrong, or a `None` check is required.

Reply:
Good observation, I'll add the `None` check